lib/endpoints/ECatMeeting.d.ts

```diff
@@ -0,0 +1,119 @@
+     * Create a new meeting
+     * @author Gabe Abrams
+     * @author Aryan Pandey
+     * @instance
+     * @memberof api.meeting
+     * @method create
+     * @param opts object containing all arguments
+     * @param opts.userId the user ID or email address of the user
+     * @param opts.meetingObj Zoom meeting object with meeting details {@link https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingcreate#request-body}}
+     * @returns Zoom meeting object {@link https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingcreate#request-body}}
+     */
+    create(opts: {
+        userId: string;
+        meetingObj: ZoomMeeting;
+    }): Promise<ZoomMeeting>;
+    /**
+     * Update a meeting
+     * @author Gabe Abrams
+     * @author Aryan Pandey
+     * @instance
+     * @memberof api.meeting
+     * @method update
+     * @param opts object containing all arguments
+     * @param opts.meetingId the Zoom ID of the meeting
+     * @param opts.meetingObj Zoom meeting object with updated details {@link https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingupdate#request-body}
+     * @param [opts.occurrenceId] ID for the meeting occurrence
+     * @returns Zoom meeting object {@link https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingcreate#request-body}}
+     */
+    update(opts: {
+        meetingId: number;
+        meetingObj: ZoomMeeting;
+        occurrenceId?: string;
+    }): Promise<ZoomMeeting>;
+    /**
+     * Delete a meeting
+     * @author Gabe Abrams
+     * @author Aryan Pandey
+     * @instance
+     * @memberof api.meeting
+     * @method delete
+     * @param opts object containing all arguments
+     * @param opts.meetingId the Zoom ID of the meeting
+     * @param [opts.occurrenceId] ID for the meeting occurrence
+     * @param [opts.notifyHosts=false] if truthy,
+     *   sends cancellation email to hosts
+     * @returns Zoom meeting object {@link https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/meetingcreate#request-body}}
+     */
+    delete(opts: {
+        meetingId: number;
+        occurrenceId?: string;
+        notifyHosts?: boolean;
+    }): Promise<ZoomMeeting>;
+    /**
+     * Get a list of ended meeting instances
+     * @author Gabe Abrams
+     * @author Aryan Pandey
+     * @instance
+     * @memberof api.meeting
+     * @method listPastInstances
+     * @param opts object containing all arguments
+     * @param opts.meetingId the Zoom ID of the meeting
+     * @returns list of ended meeting instances {@link https://marketplace.zoom.us/docs/api-reference/zoom-api/meetings/pastmeetings#responses}
+     */
+    listPastInstances(opts: {
+        meetingId: number;
+    }): Promise<ZoomMeeting[]>;
+    /**
+     * Add one alt-host if not already in the list. If another user in the alt-host
+     *   list has been deactivated, all alt-hosts are removed and the requested
+     *   user is added as the only alt-host
```

Comment:
It's a really hacky and sad fix but the best we could come up with because the alt host list is a string (comma separated) and Zoom simply says that the alt host list is invalid so we don't know which user to remove :P this is the best we could come up with so far but I'm open minded

Reply:
That's Ok. Adding that to the comment would make it more clear to other's who see thisl